src/App.jsx

```diff
@@ -14,57 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
```

Comment:
I think it would be a good idea to use the DEFAULT_VALUE constant by passing the string 'Jam' into it